### ISSUE TITLE
[Bug Report][3.1.5] props for buttons that are inside v-btn-toggle are not getting applied

### ISSUE THREAD

**Issue by peshanghiwa** [vuetifyjs/vuetify#16686]
### Environment
**Vuetify Version:** 3.1.5
**Vue Version:** 3.2.45
**Browsers:** Chrome 110.0.0.0
**OS:** Windows 10

### Steps to reproduce
Copy/Paste any example of `v-btn-toggle` component in the [official documentation](https://next.vuetifyjs.com/en/components/button-groups/) of vuetify and try to add any initial props like `color='primary'` to the `v-btn` components, and they don't work and stay transparent until you click on them and make them active

``` html
<template>
  <v-app>
    <v-main>
       <v-btn-toggle>
        <v-btn color='primary'> <!-- This color prop doesn't get applied initially until button is clicked -->
          test
        </v-btn>
 	<v-btn color='black'>
          test 2
        </v-btn>
         <v-btn variant='elevated'>
          test 3
        </v-btn>
      </v-btn-toggle>
    </v-main>
  </v-app>
</template>

<script setup>
import { ref } from 'vue'

const msg = ref('Hello World!')
</script>


```

### Expected Behavior
To have an initial styles/props applied to the buttons regardless of weather the buttons are selected or not

### Actual Behavior
The buttons stay as default and do not apply any props up until they are selected

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqVkU1uwjAQha/iepNWIrYK7QZBRXe9QReERUgMdfGfxpNUCOXutZO0CgEqVWyYNzPfPL+sT/TVOVZXgs7pAoV2KkfxkhlCFnWaO9f+bQudS9NXnbBFk6Ld71U3P9RJYZWFZeJA6hyOyWCAEBQeBwu83YgTGXa/c8ZW5cXhkkCmVxkjH3UOMje4TIQSdXhZeQU0uw3q6/NnRvE3jFh0MS34IL5Q+gKkQ+IFVrEttbOA5ERA7EhDdmA1SULuSRwurAlGtN+TZezfJ29CKUveLajyLnmI8A4XQHRCO1Yw4dintyZ8ulP0kvUNn9E5aZWohRuxzugHovNzzivjDntWWM1XocehMii1SEurVzM2ZU/PvJQehzoTXqdbsF9eQLiY0ckAjnJ3vHWgbwfuI+uxvRSRFyge+LWAFIQpBQj40/do9sz7qHfhP95sMtOELJU0Bz+KsfBthOsfa/96mJaGtYC4vYlHmgntzIVz7QLdfANaYxsE)

**Comment by yuwu9145** [comment]
- When `v-btn` is independent, `color` is applied immediately
- When `v-btn` is inside a group (e.g. `v-btn-toggle`), `color` is only applied when it is "activated". All buttons look the same when "unactivated".

This is the default behaviour from the framework

**Comment by peshanghiwa** [comment]
@yuwu9145 Ok, but what if I initially don't want the button to be white/transparent, but rather has a primary color and on active state change the color to something else.
Basicly have both ``color`` and ``active-color`` props on the component just like some other vuetify components that have these exact props.

I believe at this I should've flagged this issue as feature request and not a bug report :(

**Comment by mayakokits** [comment]
> * When `v-btn` is independent, `color` is applied immediately
> * When `v-btn` is inside a group (e.g. `v-btn-toggle`), `color` is only applied when it is "activated". All buttons look the same when "unactivated".
> 
> This is the default behaviour from the framework

@yuwu9145 The problem with this default behavior is that when a component using `v-btn-toggle` is mounted, the toggle doesn't represent the initial state of it's `v-model`. That's really counter intuitive for users. When something has a default state, as a user I want to see that.  

Now when I look at [these](https://vuetifyjs.com/en/components/button-groups/#usage) text alignment examples from the documentation, here it works as I would expect it, the initial state is visible. The examples are using `@vuetify/nightly@3.2.3-master-20230504.0` – does this mean we can expect it to work like that in v3.2.3 ? Is the change in behavior intended? :-)

**Comment by RejownAhmed** [comment]
Why isn't this fixed yet? This is deffinitely a bug

**Comment by johnleider** [comment]
> > * When `v-btn` is independent, `color` is applied immediately
> > * When `v-btn` is inside a group (e.g. `v-btn-toggle`), `color` is only applied when it is "activated". All buttons look the same when "unactivated".
> > 
> > This is the default behaviour from the framework
> 
> @yuwu9145 The problem with this default behavior is that when a component using `v-btn-toggle` is mounted, the toggle doesn't represent the initial state of it's `v-model`. That's really counter intuitive for users. When something has a default state, as a user I want to see that.
> 
> Now when I look at [these](https://vuetifyjs.com/en/components/button-groups/#usage) text alignment examples from the documentation, here it works as I would expect it, the initial state is visible. The examples are using `@vuetify/nightly@3.2.3-master-20230504.0` – does this mean we can expect it to work like that in v3.2.3 ? Is the change in behavior intended? :-)

There is no default applied to v-btn-toggle unless you use an actual v-model or mandatory="force"

https://play.vuetifyjs.com/#eNqVUctuwjAQ/BXXF1qJxCq0FwQVvfUPeiAcTLJQF7+0NqkQ4t+7TkJEoVTqzbs7OzM7Xhx4wFK8ep/XO+ATPo1gvJYRXgrL2LTOpPfNsymMVLarmnoVbRbdZqOBGWkrGR3uZwVfOyyh4D3yhGWl0w4J4FEZiXuCsAghsqlo5rcXVlqW2x4+ur1QS1TSRloBDTXdUfVb46utru5OOJ1Jzf7OVLQJTMVZMlSGEpWPLEDctQEp4x1GdmAIa3Zka3SGDSjUQYIzusSSBxM2bJYQ94M30Nqxd4e6uhs8JIGWksj4kLdsZMTnn8FZ+plDYim6QSj4hDWd1COVVBf8I0YfJkLsrN9u8tIZMaeZwJ2NykBWOTMf56P86VlUKsTzfg7BZCt0XwGQFAs+PCOPak1/9btANybex7yj7VqJ8opKEH8NmCHYChDwT98X2B/eL2ZX/pPmsbBHylIruw0XMZahiXBxsvavw4yyeUOQtpdJ5DjkrTmSaxb48htD3xjg

**Comment by johnleider** [comment]
Would it make more sense to use `v-btn-group` in this particular instance?

**Comment by yuwu9145** [comment]
Fixed in #19088, use `base-color` to apply inactive color
